test/e2e/test-che-default/test.mjs

```diff
@@ -1,6 +1,8 @@
-import { Actor } from 'apify';
+import { Actor, pushData } from 'apify';
```

Comment:
does this really work? it should not, its no longer an exported helper, its a static method of `Actor` class

Reply:
seems to be fixed in all places except this one? :]